src/Client/PullPayment.php

```diff
@@ -156,7 +156,7 @@ public function markPayoutAsPaid(
     ): bool {
         $url = $this->getApiUrl() . 'stores/' .
                     urlencode($storeId) . '/' . 'payouts/' .
-                    urlencode($payoutId);
+                    urlencode($payoutId).'/mark-paid';
```

Comment:
Very good catch, thank you. One minor nitpick, can you please add a space before and after the dot `.` so it is consistent with the other code.

Reply:
I am sorry, thanks, it's done.